frontend/src/features/workspace/components/CreateFormModal/CreateFormModalContent/CreateFormModalContent.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC, useEffect, useState } from 'react'
+import { Box, BoxProps } from '@chakra-ui/react'
+import { HTMLMotionProps, motion } from 'framer-motion'
+import { Merge } from 'type-fest'
+
+import {
+  CreateFormFlowStates,
+  useCreateFormWizard,
+} from '../CreateFormWizardContext'
+
+import { CreateFormDetailsScreen } from './CreateFormDetailsScreen'
+import { SaveSecretKeyScreen } from './SaveSecretKeyScreen'
+
+const SCREEN_ANIMATION_VARIANT = {
+  enter: (direction: number) => {
+    return {
+      x: direction > 0 ? 1000 : -1000,
```

Comment:
Just checking: are there extra restrictions on what values `direction` can take?

Reply:
no restrictions tbh; framer will automatically extrapolate. I haven't tried 1:-1 to see if it affects anything.

Reply:
Yeah, that's where I was going with my comment: if direction is a normal set (-1,1), then we can just multiply by the target position. Not very important though 😅